main.nf

```diff
@@ -70,6 +70,16 @@ def helpMessage() {
         --pon                       panel-of-normals VCF (bgzipped, indexed). See: https://software.broadinstitute.org/gatk/documentation/tooldocs/current/org_broadinstitute_hellbender_tools_walkers_mutect_CreateSomaticPanelOfNormals.php
         --pon_index                 index of pon panel-of-normals VCF
 
+    Trimming:
+        --trim_fastq                  Run Trim Galore
+        --cores                       Instructs Trim Galore to use up to 4 cores
```

Comment:
```suggestion
```

Reply:
Remove the `--cores` parameter everywhere because that value is actually calculated in the process itself using `$task.cpus`

Reply:
Also, `--cores` would be quite generic and can be confused with resources for other processes.